domain/src/main/java/org/oppia/android/domain/topic/TopicController.kt

```diff
@@ -274,6 +269,11 @@ class TopicController @Inject constructor(
         }
       }
     }
+
+    // If there is no completed chapter, it cannot be an ongoing-topic.
+    if (completedChapterProgressList.isEmpty()) {
+      return false
+    }
```

Comment:
This code isn't actually doing anything anymore (since this method is always returning false at this point). Should it be moved back up to where it was before?

Reply:
No, I added it to handle views in ProfileProgress.

Reply:
Sorry, what I mean is if you look at the logic, this if check will never change anything (the line immediately after is also returning false, so the check is completely unnecessary). That's actually why I'm wondering if this is correctly moved to this location since moving it to this location has the same logical effect as deleting it.